src/lib/inbox/inbox.ts

```diff
@@ -183,7 +234,8 @@ export class InboxTools {
       await multicall.multiCall(multicallInput, true)
 
     const firstEligibleBlockNumber =
-      currentBlockNumber.toNumber() - maxTimeVariation.delayBlocks.toNumber()
+      (currentL1BlockNumber ?? currentBlockNumber.toNumber()) -
```

Comment:
use `isParentChainArbitrum ?` instead of `currentL1BlockNumber ??`, doesn't matter too much tho